src/amo/components/Footer/index.js

```diff
@@ -76,7 +76,7 @@ export class FooterBase extends React.Component {
                 </Link>
               </li>
               <li>
-                <a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/AMO/Policy">
+                <a href="https://extensionworkshop.com/documentation/publish/add-on-policies/">
```

Comment:
We could probably use the `extensionWorkshopUrl` config parameter here (for consistency).

Reply:
good idea, is this the correct link though Will, because content is little diff on archive and this page?

Reply:
I don't know, maybe @caitmuenster would know?

Reply:
That's the correct link! 👍